cmd/notification.go

```diff
@@ -820,7 +849,8 @@ func (sys *NotificationSys) Init(objAPI ObjectLayer) error {
 		}
 		break
 	}
-	return nil
+
+	return sys.initBucketRetentionConfig(objAPI)
```

Comment:
We have to wait for quorum just like above, otherwise, we will fail while reading the retentionConfig in distributed setups during startups when all servers are not online at the same time. We shall wait for the initialization here.

Reply:
done